x-pack/plugins/alerts/server/invalidate_pending_api_keys/task.ts

```diff
@@ -125,16 +125,21 @@ function taskRunner(
             { savedObjects, http },
             { encryptedSavedObjects, security },
           ] = await coreStartServices;
+          if (!encryptedSavedObjects) {
+            throw new Error(
+              `Unable to run task because the Encrypted Saved Objects plugin is not available. Please set xpack.encryptedSavedObjects.encryptionKey in the kibana.yml or use the bin/kibana-encryption-keys command.`
+            );
+          }
```

Comment:
This shouldn't happen, right? Now that it's a required plugin it should always be there, so this is redundant code (unless I'm missing something)